paparazzi/src/test/java/app/cash/paparazzi/internal/apng/ApngWriterTest.kt

```diff
@@ -240,14 +240,6 @@ class ApngWriterTest {
     }
   }
 
-  @Test(expected = IllegalStateException::class)
-  fun throwsExceptionWhenWrittenFramesNotEqualTotalFrames() {
-    val testPath = tempFolderRule.newFile("throwsExceptionWhenWrittenFramesNotEqualTotalFrames.png").path.toPath()
-    ApngWriter(testPath, 2, 1).use { writer ->
-      writer.writeImage(createImage(squareOffset = Point(5, 5)))
-    }
-  }
```

Comment:
Does deletion of this test imply that this is no longer possible?

Reply:
Yes, we discussed the up front totalFrameCount being an opinionated API choice specific to Paparazzi, not a strict requirement for the writer itself.

Reply:
Ok, cool!